frontend/src/libs/sentry/errorReporter.ts

```diff
@@ -0,0 +1,78 @@
+import * as Sentry from "@sentry/react";
+import { typeSafeError } from "./utils/typeSafeError";
+
+interface QueryLike {
+  queryKey: readonly unknown[];
+}
+
+interface MutationLike {
+  options: {
+    mutationKey?: readonly unknown[];
+  };
+}
+
+export const sentryRenderError = (error: unknown, errorInfo: any, errorType: string) => {
```

Comment:
```suggestion
// 객체로 받기
export const sentryRenderError = (error) => {
```
사용되는 매개변수 모두 에러에 관련된 것 같은데 3개로 나눠서 받는 이유가 있으실까요??

Reply:
동의보감입니다.

Reply:
각각하는 역할의 차이점이 있어 나눠서 받게 되었습니다👍

- `error` : 실제 에러가 발생한 js 에러 객체
- `errorInfo` : 에러가 발생한 React 컴포넌트 트리의 정보
- `errorType` : 우리가 지정한 에러 태그 타입

## error 예시
```ts
{
  name: "TypeError",
  message: "Cannot read property 'name' of undefined",
  stack: "TypeError: Cannot read property 'name'..."
}
```

## errorInfo 예시
```ts
{
  componentStack: `
    at UserProfile (UserProfile.jsx:15)
    at div
    at App (App.jsx:8)
  `
}
```